liquidcore/site/settings/common.py

```diff
@@ -106,3 +106,5 @@
 DISCOVERY_URL = None
 
 INITIALIZE_LOCK_FILE_PATH = str(BASE_DIR / 'initialize.lock')
+
+LIQUID_SETUP_SUDO = False
```

Comment:
What is this and why do we need a setting here? We either have sudo on the `liquid` user for reconfiguring, or we don't. I think having a global flag for it is crazy. What's the use case here?

Reply:
Mainly so the tests pass without sudo rights. I will change the default to True and change it in `testuite/settings.py`.